content/developer/importing/porter.md

```diff
@@ -141,37 +141,84 @@ If you run into problems, your platform isn't supported, or want to reset them a
 
 #### Redirects
 
-Many legacy platform redirects can be handled by the **Redirector** plugin in the [Addons repository](https://github.com/vanilla/addons). Simply enable the plugin. Check the description for what it supports currently.
+For everything to work properly you will probably need to do 2 redirects!
+One redirect for the [path](#redirect-the-path) and one for the [domain](#redirect-the-domain).
 
-You can create custom redirects as needed using the [Routes](/developer/routes) feature in the Dashboard. Use regular expressions to match incoming URL patterns and 301 redirects to their new place. Some imports will automatically create Routes for you, so check to see.
+When setup properly, users hitting your old forum should then be
+1) globally 301 redirected to your new domain hosting Vanilla's forum which will then
+2) 301 redirect to the correct path based on the URL requested.
 
-If you run the import on localhost and then copy the database to production, be sure to manually transfer those special Routes as well.
+##### Redirect the path
 
-Once you have correct redirects set in Vanilla, you must set up a global redirect from your old forum address to Vanilla. See the `forum-redirector` folder in the [Addons repository](https://github.com/vanilla/addons) for help setting this up. This is *not* the same as the Redirector plugin above (which is at `plugins/Redirector`).
+The "path" is everything that comes after the domain.
+In `http://domain.com/forum/thread.php?id=10&pageNumber=2` that would be `/forum/thread.php?id=10&pageNumber=2`
```

Comment:
You realize that this line break is only visible in the source?
It is much easier to read these, when disposed like so, in the editor.